Opus demo ram improvements

interfer opened the pull request:
**Once FreeRTOS merged, N22 bin should be updated and then will be ready for review**

andriy-bilynskyy commented:
I think due to overlays n22 side also should be changed. So new revision of n22 is required.

interfer committed:
opus: ram: freed D25 DLM for N22 usage

Passed unused RAM for N22 usage.
Increased IPC msg buf for 48KHz config.
Added as DT overlay for W91 Opus app.

Updated N22 binary revision

Signed-off-by: Dmytro Huz <diman1436@gmail.com>

opus: changed log level of Opus demo app

Now it's default INFO for better timing.

Signed-off-by: Dmytro Huz <diman1436@gmail.com>
